doc/classes/OS.xml

```diff
@@ -117,7 +117,7 @@
 			<return type="PoolStringArray">
 			</return>
 			<description>
-				Return the commandline passed to the engine.
+				Returns the command line arguments.
```

Comment:
Sorry - I didn't mean remove "passed to the engine." I was just referring to the part of the sentence that needed changing.